### ISSUE TITLE
Please add support for YAML configuration again

### ISSUE THREAD

**Issue by 1RandomDev** [rospogrigio/localtuya#1020]
> removed YAML configuration (since it's being deprecated almost everywhere)

Plsss add support for configuring devices via YAML again. It was so much easier to see all settings in one file without having to click through menus and you also could just copy the code when setting up many devices at once.
I see that (unfortunately) more and more new integrations only can be configured via the which is okay for things where you just have to enter a url or password, but for a integration that requires so much configuration (in my case at least 5 datapoints per device) it would just be much more convenient to type it all in a file.

I also already tried to find the place where the GUI editor stores it's settings, but it seems that for this integration it's not in `.storage`.

**Comment by btehan** [comment]
I tried upgrading to 4.1.0 and it broke a device that I have setup with a YAML file.  I haven't had time to troubleshoot the issue so I rolled back via a HAOS backup.  ¯\_(ツ)_/¯

**Comment by findel** [comment]
Yes please! I much prefer configuration in code that I can track changes on etc.

**Comment by ingoratsdorf** [comment]
Yes, please. YAML was way easier, could be archived, had a good overview and new devices could just be copied in seconds.

**Comment by osnwt** [comment]
Yes, please! I use UI only when no other way or for experimenting with new devices/integrations. For other purposes a text file is much better.

BTW, what was the reason to remove it at all?

**Comment by mat-sop** [comment]
I'm new to home assistant and localtuya so this question might seem dumb. Is there any other way to import/export/backup configuration other than removed yaml files?

Right now I'm experimenting with docker image and some devices I already have, but eventually I'll want to migrate to stand-alone server. Right now I can't see other option than screenshots and clicking the same stuff in ui. If I understand correctly the same will happen whenever I'll need to reconnect my devices, like when I change my wifi ssid?
